crates/forge/bin/cmd/bind.rs

```
@@ -239,13 +281,29 @@ impl BindArgs {
         Ok(())
     }
 
+    fn check_alloy(&self, artifacts: &Path, bindings_root: &Path) -> Result<()> {
+        let mut bindings = self.get_solmacrogen(artifacts)?;
+        bindings.generate_bindings()?;
+        println!("Checking bindings for {} contracts", bindings.instances.len());
+        let _ = bindings.check_consistency(
```

the error is ignored